include/proxy/ProxySession.h

```diff
@@ -111,6 +111,9 @@ class ProxySession : public VConnection, public PluginUserArgs<TS_USER_ARGS_SSN>
 
   virtual PoolableSession *get_server_session() const;
 
+  virtual uint64_t is_protocol_framed() const;
```

Comment:
Why does this return `uintr64_t` instead of `bool`, or alternatively why is it named "is_..." when that should indicate a boolean return?

Reply:
Changed it to bool.